CODEOWNERS

```diff
@@ -43,6 +43,5 @@
 /src/cpp/ @kroening @tautschnig @peterschrammel
 
 
-/scripts/ @diffblue/devops @thk123 @forejtv @peterschrammel
-/.travis.yml @diffblue/devops @thk123 @forejtv @peterschrammel
-/appveyor.yml @diffblue/devops @thk123 @forejtv @peterschrammel
+# All remaining files are assumed to be low-risk
+# Any two reviewers can be assigned
```

Comment:
To be honest, I wouldn't really consider any of the CI configuration files "low risk." Changes to them may have impact on the bills paid. Indeed the CodeBuild configuration files should be added here.  (I do agree that `/scripts/` is low risk.)